fabric-renderer-api-v1/src/client/java/net/fabricmc/fabric/api/renderer/v1/mesh/QuadView.java

```diff
@@ -201,7 +201,7 @@ default Vector2f copyUv(int vertexIndex, @Nullable Vector2f target) {
 	 * @param target Target array for the baked quad data.
 	 *
 	 * @param targetIndex Starting position in target array - array must have
-	 * at least 28 elements available at this index.
+	 * at least {@link #VANILLA_QUAD_STRIDE} elements available at this index.
```

Comment:
You can even use `@value` here

Reply:
IntelliJ says `@value tag must reference a field with a constant initializer`, so `@link` is a better option.

Reply:
Right...